source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Infrastructure.Core/Function/HttpResponseBuilderTests.cs

```diff
@@ -77,6 +78,32 @@ public void CreateAcceptedResponse_Creates_AcceptedResponseWithCorrelationId(Fun
             responseData.StatusCode.Should().Be(HttpStatusCode.BadRequest);
         }
 
+        [Theory]
+        [InlineAutoMoqData]
+        public void CreateBadRequestResponseWithText_Creates_Response_With_Text_In_Xml_Body(
```

Comment:
Elsewhere in the codebase we seek to name test-methods using the Gherkin syntax (Given_When_Then) and i suggest: 
```suggestion
        public void CreateBadRequestResponseWithText_WhenBadRequest_CreatesResponseWithTextInXmlBody(
```